.github/workflows/code-health.yml

```diff
@@ -89,11 +84,11 @@ jobs:
       - name: Run ShellCheck
         uses: bewuethr/shellcheck-action@v2
   call-acceptance-tests-workflow:
-    needs: [build, lint, shellcheck, unit-test, website-lint]
+    needs: [build, shellcheck, unit-test, website-lint]
```

Comment:
why did you remove lint here? if the lint fails we should not run the test

Reply:
the lint is now running inside build, so threre is still a dependency on it